Dockerfile

```diff
@@ -6,38 +6,26 @@ WORKDIR /app
 
 COPY ./package.json .
 
+COPY .env . 
+
 COPY ./ .
 
 RUN npm install && npm run build
 
 #### Stage 2: Serve the application from Nginx
 
-FROM nginx:1.24.0-alpine3.17
-
-RUN apk update && apk add --no-cache bash
-
-RUN  addgroup nginx nginx
+FROM nginxinc/nginx-unprivileged:1.25.1-perl
 
 # Nginx config
 RUN rm -rf /etc/nginx/conf.d
-
+USER root 
```

Comment:
One recommendation to try, that could be better than declaring the root user is to add the `pid /tmp/nginx.pid` line to the nginx conf. Please have a look at these documentation:

- https://hub.docker.com/_/nginx -> Running nginx as a non-root user
- https://hub.docker.com/r/nginxinc/nginx-unprivileged -> Common Issues